src/parser/Parser.ts

```diff
@@ -1056,7 +1056,14 @@ export class Parser {
                 | Expr.Expression
                 | Stmt.PrintSeparator.Tab
                 | Stmt.PrintSeparator.Space)[] = [];
-            values.push(expression());
+
+            //print statements can be empty, so look for empty print conditions
+            if (isAtEnd() || check(Lexeme.Newline, Lexeme.Colon)) {
+                var emptyStringLiteral = new Expr.Literal(new BrsString(""), printKeyword.location);
```

Comment:
Mind switching this to a `let` real quick?  I could have sworn I had `tslint` configured to disallow `var` 🤔

Reply:
@sjbarag done.